datasets/HakaiQuadraBoLResearch.xml

```diff
@@ -1,21 +1,22 @@
+
 <dataset type="EDDTableFromAsciiFiles" datasetID="HakaiQuadraBoLResearch" active="true">
-	<iso19115File>https://pac-dev1.cioos.org/dev/metadata/pacific/high_resolution_record_of_surf_a6f6c.xml</iso19115File>
-	<reloadEveryNMinutes>10080</reloadEveryNMinutes>
-	<updateEveryNMillis>10000</updateEveryNMillis>
-	<fileDir>/datasets/HakaiQuadraBoLResearch/</fileDir>
-	<fileNameRegex>.*\_reformated\.csv</fileNameRegex>
-	<recursive>true</recursive>
-	<pathRegex>.*</pathRegex>
-	<metadataFrom>last</metadataFrom>
-	<standardizeWhat>0</standardizeWhat>
-	<charset>UTF-8</charset>
-	<columnSeparator>,</columnSeparator>
-	<columnNamesRow>1</columnNamesRow>
-	<firstDataRow>2</firstDataRow>
-	<sortedColumnSourceName></sortedColumnSourceName>
-	<sortFilesBySourceNames></sortFilesBySourceNames>
-	<fileTableInMemory>false</fileTableInMemory>
-	<accessibleViaFiles>true</accessibleViaFiles>
+    <iso19115File>https://pac-dev1.cioos.org/dev/metadata/pacific/high_resolution_record_of_surf_a6f6c.xml</iso19115File>
+    <reloadEveryNMinutes>10080</reloadEveryNMinutes>
+    <updateEveryNMillis>10000</updateEveryNMillis>
+    <fileDir>/datasets/HakaiQuadraBoLResearch/</fileDir>
+    <fileNameRegex>.*\.csv</fileNameRegex>
+    <recursive>false</recursive>
+    <pathRegex>.*</pathRegex>
+    <metadataFrom>last</metadataFrom>
+    <standardizeWhat>0</standardizeWhat>
+    <charset>UTF-8</charset>
+    <columnSeparator>,</columnSeparator>
+    <columnNamesRow>7</columnNamesRow>
+    <firstDataRow>8</firstDataRow>
+    <sortedColumnSourceName></sortedColumnSourceName>
```

Comment:
Not sure if these empty tags (sortedColumnSourceName,sortFilesBySourceNames) have any effect, but could probably remove them